plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplate.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * WooCommerce Product Block Editor
+ */
+
+namespace Automattic\WooCommerce\Internal\Admin\Features\ProductBlockEditor;
```

Comment:
Since 3PDs will need to use this class to register new product templates, it should not be in an `Internal` namespace.

It probably belongs somewhere like `Automattic\WooCommerce\Admin\Features\ProductBlockEditor`

Reply:
done -> https://github.com/woocommerce/woocommerce/pull/41823/commits/8ed3109d73e2156ee60ee39a7d19c379e435d358